### ISSUE TITLE
Mirage archer damage calculation

### ISSUE THREAD

**Issue by deathbeam** [PathOfBuildingCommunity/PathOfBuilding#1511]
**Is your feature request related to a problem? Please describe.**
I would like to see Mirage Archer being suggested when choosing support gems based on its contribution to my damage and I would like to also see it having impact on my damage when looking at it in damage overview.

**Describe the solution you'd like**
Mirage Archer should be taken into account for skills that are supported by it and also when listing and sorting available support gems. The left side could display something like:

```
Mirage Archer DPS:
Mirager Archer <insert damage type here, for example ignite> DPS:
Total DPS including mirage archer:
```

This is only how I would see it implemented, but if there is better solution feel free to comment it below. See https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/976 for feature request requesting same thing for Saviour Clone DPS.

**Describe alternatives you've considered**
Alternatively this could be implemented as separate entity, but this will probably break the sorting use case for support gems and also not being able to see the combined DPS.

**Additional context**
See wiki for Mirage Archer here, it describes its mechanics and also approximate damage contribution: https://pathofexile.gamepedia.com/Mirage_Archer_Support

**Comment by deathbeam** [comment]
Oh forgot to mention, here is the original issue that was closed and where it was suggested for me to reword the request, what I did: https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues/1279

**Comment by onefuncman** [comment]
This is even more relevant now that we have +2 Mirage Archer ascendency and this is seeing play in Toxic Rain builds like this one: https://pastebin.com/B7hFuCHC (from https://www.pathofexile.com/forum/view-thread/3009514 )

The forum post says:
>  Each mirage archer is like 35.6% damage multiplier 

but this is unsourced and needs to be validated.

**Comment by Regisle** [comment]
This is on my todo list, but might be a while before its in an official release

**Comment by onefuncman** [comment]
I found https://docs.google.com/spreadsheets/d/1GyF3XY0qh9Of5re9e6GpSJWu6tvQrZlOqYMqnazjwdA/edit#gid=0 from https://www.reddit.com/r/pathofexile/comments/igs0u7/toxic_rain_new_calculator_spreadsheet_includes/ which provides sourcing for the Mirage Archer calculations which could be used as the basis for completing this issue.

**Comment by Nostrademous** [comment]
Fixed in #2294

**Comment by Nostrademous** [comment]
Implemented in 2.0.0
